### ISSUE TITLE
does not work on flatpack / snap installations

### ISSUE THREAD

**Issue by jerger** [gopasspw/gopass-jsonapi#82]
### Summary
while new ubuntu head toward firefox as snap (unfortunately) gopass-jsonapi does not work on flatpack / snap installations

### Steps To Reproduce
1. sudo dpkg -i gopass-jsonapi_1.11.1_linux_amd64.deb
2. gopass-jsonapi configure
3. choose firefox
4. try out gopass-bridge on firefox -> shows no connection

### Environment
vanilla xubuntu22.04

### Additional context
I assume, storing native messaging host file at `.mozilla/native-messaging-hosts/com.justwatch.gopass.json` is not the correct location for snap / flatpack?

**Comment by dominikschulz** [comment]
Flatpack and snap seem like a terrible idea to me.

At least in their current form.

IIUC using native extensions with snap is not yet possible at all.

**Comment by jerger** [comment]
Yes I totally agree, but while using ubuntu there is no really good alternative ...

**Comment by dominikschulz** [comment]
IIUC this does affect other extensions as well and so far they don't seem to have a good solution (but personally I'm not following closely as one can e.g. just get Chrome as a `deb`).

Any ideas are very welcome.

**Comment by skewballfox** [comment]
flatpak side, I don't believe it's currently possible. [Here's a relevant PR](https://github.com/flatpak/xdg-desktop-portal/pull/705) in the works that could change that
